com.unity.render-pipelines.core/ShaderLibrary/Color.hlsl

```diff
@@ -254,7 +254,11 @@ real YCoCgCheckBoardEdgeFilter(real centerLum, real2 a0, real2 a1, real2 a2, rea
 }
 
 // Converts linear RGB to LMS
+#if defined(SHADER_API_SWITCH) // Must use full float precision on Switch to avoid artefact when using ACES tonemapping
+float3 LinearToLMS(float3 x)
+#else
 real3 LinearToLMS(real3 x)
+#endif
```

Comment:
Imo if this function needs full precision it should be changed for all platforms, otherwise we have the same issue on mobile. Checking the original PR, @eh-unity made the same comment.

Reply:
I tested on iPhone8 metal and Galaxy S9 Mali-G72 Vulkan but could not reproduce the issue, hence why it is conservatively kept it Switch only. But I can enable it for all platforms if you prefer. There is similar floating point fix in `AcesTonemap()` and `NeutralCurve()` for Switch but done from a different PR, do you want to change them for all platforms too?